com/win32com/client/dynamic.py

```diff
@@ -64,16 +64,11 @@ def debug_attr_print(*args):
         print()
 
 
-def MakeMethod(func, inst, cls):
-    return types.MethodType(func, inst)
```

Comment:
This really should've been removed with https://github.com/mhammond/pywin32/commit/9a11ef04da2a45d339cf8eb582c7dc12edb327fd